src/test/java/org/kohsuke/github/CommitTest.java

```diff
@@ -120,4 +120,13 @@ public void commitSignatureVerification() throws Exception {
                     commit.getCommitShortInfo().getVerification().getPayload());
         }
     }
+
+    @Test // issue 883
+    public void commitDateNotNull() throws Exception {
+        GHRepository repo = gitHub.getRepository("hub4j/github-api");
+        GHCommit commit = repo.getCommit("ed4f9c8176866977677c99ac9668a8ce10231bc8");
+
+        assertNotNull(commit.getCommitShortInfo().getAuthoredDate());
```

Comment:
Please check for more than just not `null`.  There is a known expected value that isn't going to change over time.